src/main/java/rip/bolt/ingame/commands/providers/TournamentProvider.java

```diff
@@ -23,28 +27,26 @@ public TournamentProvider(TournamentManager tournamentManager) {
     }
 
     @Override
-    public boolean isProvided() {
-        return true;
+    public String getName() {
+        return "Tournament format";
     }
 
     @Override
     public TournamentFormat get(CommandSender commandSender, CommandArgs commandArgs, List<? extends Annotation> list) throws ArgumentException, ProvisionException {
-        Optional<TournamentFormat> tournamentFormat = tournamentManager.currentTournament();
```

Comment:
How does this not break nearly everything? There's a lot of logic here that being deleted will break things

Reply:
That's an unused intake provider, wich wasn't being used in any command

Reply:
@Pablete1234 https://github.com/PGMDev/Events/blob/master/src/main/java/rip/bolt/ingame/commands/TournamentUserCommands.java#L41?